packages/itwinui-css/src/button/index.scss

```diff
@@ -2,11 +2,8 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './button';
 @import './default';
-@import './cta';
-@import './high-visibility';
+@import './borderless';
+@import './status-color';
```

Comment:
So I'm getting a little confused by these file names, particularly `button`, `default` and `status-color`.

Is there a reason we need to keep `button` and `default` separate? I feel like the default styling should be part of the base button.

Also, in another comment I suggested renaming `status-color` to `variants`.

Reply:
Also where did `button-icon` disappear?